examples/systemcalls/systemcalls.c

```diff
@@ -16,8 +22,14 @@ bool do_system(const char *cmd)
  *   and return a boolean true if the system() call completed with success 
  *   or false() if it returned a failure
 */
+    int result = system(cmd);
 
-    return true;
+    if(result <= 0)
```

Comment:
@dwalkes can you please review this? will result = 0 should return as failure of system call?